candi/bashible/common-steps/node-group/069_start_kubelet.sh.tpl

```diff
@@ -66,20 +66,12 @@ fi
 {{- end }}
 
 if bb-flag? kubelet-need-restart; then
-{{- if ne .runType "ImageBuilding" }}
-  bb-log-warning "'kubelet-need-restart' flag was set. Kubelet should be restarted!"
-  {{ if eq .runType "ClusterBootstrap" }}
-  bb-log-info "Restart kubelet service..."
 
+{{- if ne .runType "ImageBuilding" }}
+  bb-log-warning "'kubelet-need-restart' flag was set. Restart kubelet."
   systemctl restart "kubelet.service"
-
-  bb-log-info "Kubelet service was restarted."
-  {{ else }}
-  if ! bb-flag? reboot; then
-    bb-log-info "Restart kubelet service..."
-
-    systemctl restart "kubelet.service"
-
+  {{ if ne .runType "ClusterBootstrap" }}
+  if [[ "${FIRST_BASHIBLE_RUN}" != "yes" ]] && ! bb-flag? reboot; then
```

Comment:
The else-message is weird.

>      bb-log-info "Skip restarting kubelet because node will be rebooted."

kubelet has been restarted above, i suppose, we need to remove "else" at all.

Reply:
yep, i'll fix that